images/openshift-enterprise-hyperkube.yml

```diff
@@ -9,6 +9,10 @@ content:
       branch:
         target: release-{MAJOR}.{MINOR}
       url: git@github.com:openshift-priv/kubernetes.git
+    modifications:
+    - action: replace
+      match: "AS builder"
+      replacement: "AS builder\nENV GOMAXPROCS=16"
```

Comment:
In my testing, even 16 was hit and miss (one failure, one success).  8 should be safe.